client/src/app/containers/Game/index.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+
+import { Grid, IconButton } from '@material-ui/core';
+import RefreshIcon from '@material-ui/icons/Refresh';
+
+import { useAuth } from '../../../auth/useAuth';
+import ActionBox from './ActionBox';
+import CribHand from './CribHand';
+import { Game, Phase } from './models';
+import PlayerHand from './PlayerHand';
+import PlayingCard from './PlayingCard';
+import ScoreBoard from './ScoreBoard';
+import { useGame } from './useGame';
+
+const showCutCard = (phase: Phase) =>
+    !['Deal', 'BuildCrib', 'Cut'].includes(phase);
```

Comment:
:mag: could we choose the inclusive case instead? something like `['Pegging', 'CountHands', 'CountCrib'].includes(phase);`?